lib/Backend/BailOut.cpp

```diff
@@ -252,10 +251,19 @@ BailOutInfo::FinalizeBailOutRecord(Func * func)
             }
             // The locals size contains the inlined-arg-area size, so remove the inlined-arg-area size from the
             // adjustment for normal locals whose offsets are relative to the start of the locals area.
-            offset -= (localsSize - inlineeArgStackSize);
+            offset -= (inlinedArgSlotAdjust - inlineeArgStackSize);
```

Comment:
Well  this is in the MD_GROW_LOCALS_AREA_UP
`uint inlinedArgSlotAdjust = (func->m_localStackHeight + func->m_ArgumentsOffset);`
`int localsSize = func->m_localStackHeight + func->m_ArgumentsOffset;`

Why can't they be substituted?